source/chapters/appendix.rst

```diff
@@ -296,6 +296,12 @@ file(s) on the command line. These will be loaded at start-up.  Each file you
 specify will be loaded into the next virtual deck. For a list of supported file
 types, go to :ref:`configuration-import`.
 
+To launch Mixxx using the command line:
```

Comment:
I think we should name the whole section "Launching Mixxx from the Command Line", then make "Command Line Options" a subsection of it.